task.sql

```diff
@@ -1,17 +1,38 @@
-CREATE DATABASE ShopDB; 
-USE ShopDB; 
-
--- Create a table to store countries 
+CREATE DATABASE ShopDB;
+USE ShopDB;
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 ) ENGINE=InnoDB;
 
--- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIPCache (
+    ID INT AUTO_INCREMENT,
+    IPRange VARCHAR(100),
```

Comment:
The `GeoIPCache` table uses the `Memory` storage engine, which does not support `TEXT` data types. Consider changing the `IPRange` column to a `VARCHAR` type with an appropriate length.